vault-plugins/flant_iam/usecase/group_test.go

```diff
@@ -3,73 +3,25 @@ package usecase
 import (
 	"testing"
 
+	"github.com/stretchr/testify/assert"
+
+	"github.com/flant/negentropy/vault-plugins/flant_iam/fixtures"
 	"github.com/flant/negentropy/vault-plugins/flant_iam/model"
 	"github.com/flant/negentropy/vault-plugins/flant_iam/uuid"
 	"github.com/flant/negentropy/vault-plugins/shared/io"
 )
 
-const (
-	groupUUID1 = "00000000-0001-0000-0000-000000000000"
-	groupUUID2 = "00000000-0002-0000-0000-000000000000"
-	groupUUID3 = "00000000-0003-0000-0000-000000000000"
-	groupUUID4 = "00000000-0004-0000-0000-000000000000"
-	groupUUID5 = "00000000-0005-0000-0000-000000000000"
-)
-
-var (
-	group1 = model.Group{
-		UUID:            groupUUID1,
-		TenantUUID:      tenantUUID1,
-		Identifier:      "group1",
-		Users:           []string{userUUID2, userUUID3},
-		Groups:          []string{groupUUID3},
-		ServiceAccounts: []string{serviceAccountUUID1},
-		Origin:          model.OriginIAM,
-	}
-	group2 = model.Group{
-		UUID:       groupUUID2,
-		TenantUUID: tenantUUID1,
-		Identifier: "group2",
-		Users:      []string{userUUID1, userUUID3},
-		Origin:     model.OriginIAM,
-	}
-	group3 = model.Group{
-		UUID:            groupUUID3,
-		TenantUUID:      tenantUUID2,
-		Identifier:      "group3",
-		Users:           []string{userUUID3, userUUID4},
-		ServiceAccounts: []string{serviceAccountUUID1},
-		Origin:          model.OriginIAM,
-	}
-	group4 = model.Group{
-		UUID:            groupUUID4,
-		TenantUUID:      tenantUUID1,
-		Identifier:      "group4",
-		Users:           []string{userUUID2, userUUID3},
-		Groups:          []string{groupUUID2, groupUUID3},
-		ServiceAccounts: []string{serviceAccountUUID2, serviceAccountUUID3},
-		Origin:          model.OriginIAM,
-	}
-	group5 = model.Group{
-		UUID:       groupUUID5,
-		TenantUUID: tenantUUID1,
-		Identifier: "group5",
-		Groups:     []string{groupUUID2, groupUUID1},
-		Origin:     model.OriginIAM,
-	}
-)
-
 func createGroups(t *testing.T, repo *model.GroupRepository, groups ...model.Group) {
 	for _, group := range groups {
 		tmp := group
 		tmp.FullIdentifier = uuid.New()
 		err := repo.Create(&tmp)
-		dieOnErr(t, err)
+		assert.NoError(t, err)
```

Comment:
Helpers in the `assert` package return bool value. As I can see, this value is not used, so I suggest using the `require` package instead.